android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -201,10 +201,6 @@ protected WebView createViewInstance(ThemedReactContext reactContext) {
 
     settings.setAllowFileAccess(false);
     settings.setAllowContentAccess(false);
-    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
-      settings.setAllowFileAccessFromFileURLs(false);
-      setAllowUniversalAccessFromFileURLs(webView, false);
-    }
```

Comment:
You're gonna want to keep these 😅

Reply:
is there a place where we can just set to `false` regardless

Reply:
It's fine to keep them here I believe

Reply:
i just meant: do we need the `if`?

Reply:
I assume this if-clause is here only because support for this function was only introduced from jelly bean and onwards, so I'd say yes.

Reply:
Btw- I think the you can tree-shake a bunch of conditionals like this if you change the target for the build to the ones we support. 

I know that was the case for iOS, so should be the same for Android.